diagnostic.md

```diff
@@ -48,11 +54,11 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    The value can be referenced as movie.movie_id.
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+    Assign the data to a variable and pass the variable to a component.
```

Comment:
Yes, that is one way of doing it. I think what were were looking for is by using `{{model}}` in the template.
👍